src/widget/wtracktableview.h

```diff
@@ -38,6 +38,8 @@ class WTrackTableView : public WLibraryTableView {
     void onShow() override;
     bool hasFocus() const override;
     void keyPressEvent(QKeyEvent* event) override;
+    void loadSelectedTrack() override;
+    void loadSelectedTrackToGroup(QString group, bool play = false) override;
```

Comment:
Is the default value for the last parameter still needed? I suggest to remove it, default parameters for polymorphic methods are confusing. The base class doesn't define a default value.

Reply:
Earlier loadSelectedTrackToGroup` was implemented as:
```cpp
public:
  void loadSelectedTrackToGroup(QString group, bool play) override;
private slots:
  void loadSelectionToGroup(QString group, bool play = false);
.
.
void WTrackTableView::loadSelectedTrackToGroup(QString group, bool play) {
    loadSelectionToGroup(group, play);
}
```
I have removed the `loadSelectionToGroup` function and moved it's implementation to the override `loadSelectedTrackToGroup` which added the default parameter.
Should I add back `loadSelectionToGroup` as well?

Reply:
No, those functions are already in WTrackMenu and would be unused in WTrackTableView. Just remove the default parameter here.